frontend/pages/entry/new.tsx

```diff
@@ -0,0 +1,95 @@
+import { FileUpload } from '@mui/icons-material'
+import { Button, Card, Container, Divider, Stack, Typography } from '@mui/material'
+import { useRouter } from 'next/router'
+import Title from 'src/common/Title'
+import CreateEntry from 'src/entry/CreateEntry'
+import { EntryKind, EntryKindKeys, isEntryKind } from 'types/types'
+
+export default function NewEntry() {
+  const router = useRouter()
+  const { kind }: { kind?: string } = router.query
+
+  const handleSelectEntryKind = (value: EntryKindKeys) => {
+    router.replace({
+      query: { ...router.query, kind: value },
+    })
+  }
+
+  return (
+    <>
+      <Title text='Create data card or model' />
+      <Container maxWidth='md'>
+        {isEntryKind(kind) ? (
+          <CreateEntry kind={kind} />
+        ) : (
+          <Card sx={{ p: 4 }}>
+            <Stack spacing={4} justifyContent='center' alignItems='center'>
+              <Typography variant='h6' component='h1' color='primary'>
+                Create Data Card/Model
+              </Typography>
+              <FileUpload color='primary' fontSize='large' />
+              <Stack
+                direction={{ sm: 'column', md: 'row' }}
+                spacing={4}
```

Comment:
Minor and I appreciate this wasn't you originally, but is it worth using `theme.spacing` here? Good opportunity to improve older code

Reply:
This shorthand uses `theme.spacing` under the hood. I've instead removed all unnecessary uses of `theme.spacing()` with the shorthand.